### PR TITLE
front: clean scenario page

### LINKED ISSUE #9407: front: clean scenario page
Some small improvements to the scenario page:
- change the background color of all the simulation-results components
- change the map style in simulation results (border-radius, margins, box-shadow, height)
- adjust some spacing between some components
- clean a bit the html & the css classes

### PR DESCRIPTION
**Pull request by clarani**
closes #9407 

Changes
- background color of the main components of SimulationResults
- style of the map: add border radius and box-shadow, remove useless divs
- small improvements on the margins

Before:
image
image
image

After:
image
image
image

### CODE REVIEW

**Review by theocrsb — COMMENTED** [review]
I don't know if this needs to be done in this PR, but we could also swap the position between the output table and the map.

File: front/src/styles/scss/applications/operationalStudies/_simulationresults.scss [hunk 1]
```diff
@@ -39,15 +39,24 @@
...
     &.speedspacechart-container {
+      display: flex;
       background-color: rgb(247, 246, 238);
+      margin-bottom: 28px;
     }
   }
 
+  .simulation-map {
+    position: relative;
+    height: 450px;
```
Comment (theocrsb) [inline front/src/styles/scss/applications/operationalStudies/_simulationresults.scss]: On the mockup, the height is 750px. I can't find the 450px.

File: front/src/styles/scss/applications/operationalStudies/_simulationresults.scss [hunk 2]
```diff
@@ -39,15 +39,24 @@
...
+    margin: 0 30px 56px 30px;
```
Comment (theocrsb) [inline front/src/styles/scss/applications/operationalStudies/_simulationresults.scss]:
```suggestion
    margin: 0 30px 56px 28px;
```

File: front/src/styles/scss/applications/operationalStudies/_simulationresults.scss [hunk 3]
```diff
@@ -39,15 +39,24 @@
...
+    border-radius: 20px;
```
Comment (theocrsb) [inline front/src/styles/scss/applications/operationalStudies/_simulationresults.scss]:
```suggestion
    border-radius: 10px;
```

File: front/src/styles/scss/applications/operationalStudies/_simulationresults.scss [hunk 4]
```diff
@@ -39,15 +39,24 @@
 
   .osrd-simulation-container {
     position: relative;
-    background-color: white;
     padding: 1rem;
     border-radius: 4px;
 
     &.speedspacechart-container {
+      display: flex;
       background-color: rgb(247, 246, 238);
```
Comment (SharglutDev) [inline front/src/styles/scss/applications/operationalStudies/_simulationresults.scss]: We can/should use `ambiantB10` from `_variables.scss`.
And maybe we could set the background-color globally for all simulation-results so we don't have to use it here and maybe in other places ?

File: front/src/styles/scss/applications/operationalStudies/_simulationresults.scss [hunk 5]
```diff
@@ -39,15 +39,24 @@
...
+      margin-bottom: 28px;
     }
   }
 
+  .simulation-map {
+    position: relative;
+    height: 450px;
+    margin: 0 30px 56px 30px;
+    border-radius: 20px;
+    box-shadow: 0 5px 10px rgba(0, 0, 0, 0.3);
```
Comment (SharglutDev) [inline front/src/styles/scss/applications/operationalStudies/_simulationresults.scss]: If I copy the css for the map in the [mockup](https://www.sketch.com/s/2fb064cd-c483-46ac-bd30-72705fd63e75/p/9559F6FC-69EE-4DB8-BE1A-77F60F10C97F/canvas#Inspect), I get this for the box-shadows : 

```suggestion
    box-shadow: 0px 1px 0px rgba(255, 255, 255, 1) inset, 0px 4px 7px -3px rgba(255, 171, 88, 0.17), 0px 2px 4px rgba(0, 0, 0, 0.22);
```

File: front/src/styles/scss/applications/operationalStudies/_simulationresults.scss (hunk 1 above)
Comment (SharglutDev) [inline front/src/styles/scss/applications/operationalStudies/_simulationresults.scss]: On the mockup (page 665 in Scenario), I see this for the height : 

image

Other question : we start using px expect for font-size for now on ?

File: front/src/styles/scss/applications/operationalStudies/_simulationresults.scss (hunk 3 above)
Comment (SharglutDev) [inline front/src/styles/scss/applications/operationalStudies/_simulationresults.scss]: Seems to be 10px on mockup

File: front/src/styles/scss/applications/operationalStudies/_simulationresults.scss (hunk 2 above)
Comment (SharglutDev) [inline front/src/styles/scss/applications/operationalStudies/_simulationresults.scss]: There should be 48px on the left of the map (10px from global page padding and 38 here)

```suggestion
    margin: 0 30px 56px 38px;
```

File: front/src/applications/operationalStudies/views/SimulationResults.tsx
Comment (SharglutDev) [inline front/src/applications/operationalStudies/views/SimulationResults.tsx:1]: There are some unused imports/const in the file

**Review by SharglutDev — COMMENTED** [review]
Thank you for the PR. As there is nothing in the issue, not sure what you planned to fix here and if my comments are pertinent. But as you started to change some styles, I followed your lead :)

File: front/src/applications/operationalStudies/views/SimulationResults.tsx
Comment (RomainValls) [reply front/src/applications/operationalStudies/views/SimulationResults.tsx:1]: MAP_MIN_HEIGHT / useRef

File: front/src/styles/scss/applications/operationalStudies/_simulationresults.scss (hunk 4 above)
Comment (clarani) [reply front/src/styles/scss/applications/operationalStudies/_simulationresults.scss]: I removed this background color, it was not used actually

File: front/src/styles/scss/applications/operationalStudies/_simulationresults.scss (hunk 1 above)
Comment (clarani) [reply front/src/styles/scss/applications/operationalStudies/_simulationresults.scss]: Done :) I think we can start using px :+1:

File: front/src/applications/operationalStudies/views/SimulationResults.tsx
Comment (clarani) [reply front/src/applications/operationalStudies/views/SimulationResults.tsx:1]: Thanks, done !

File: front/src/styles/scss/applications/operationalStudies/_simulationresults.scss [hunk 6]
```diff
@@ -39,15 +39,22 @@
 
   .osrd-simulation-container {
     position: relative;
-    background-color: white;
     padding: 1rem;
     border-radius: 4px;
 
     &.speedspacechart-container {
-      background-color: rgb(247, 246, 238);
+      margin-bottom: 28px;
```
Comment (SharglutDev) [inline front/src/styles/scss/applications/operationalStudies/_simulationresults.scss]: The margin is not quite this one on mockup : 

image

We have this right now : 

image

File: front/src/applications/operationalStudies/views/SimulationResults.tsx [hunk 7]
```diff
@@ -198,13 +195,30 @@ const SimulationResults = ({
...
+      </div>
+
       {/* TIME STOPS TABLE */}
       {selectedTrainSchedule &&
         trainSimulation.status === 'success' &&
         pathProperties &&
         operationalPoints &&
         infraId && (
-          <div className="osrd-simulation-container mb-2">
+          <div className="osrd-simulation-container mb-4">
```
Comment (SharglutDev) [inline front/src/applications/operationalStudies/views/SimulationResults.tsx]: Margin below the output table is like this : 

image

Comment (clarani) [reply front/src/applications/operationalStudies/views/SimulationResults.tsx]: I think it's done :white_check_mark:

File: front/src/styles/scss/applications/operationalStudies/_simulationresults.scss (hunk 6 above)
Comment (clarani) [reply front/src/styles/scss/applications/operationalStudies/_simulationresults.scss]: Done :white_check_mark:

**Comment by theocrsb** [comment]
The left spacing of the map is good but now we need to align all item with it 🙂
image

File: front/src/applications/operationalStudies/views/SimulationResults.tsx (hunk 7 above)
Comment (SharglutDev) [reply front/src/applications/operationalStudies/views/SimulationResults.tsx]: Hmm sorry mb-2 is doing 8px but we need 11px to match the mockup :/

**Review by theocrsb — APPROVED** [review]

**Review by SharglutDev — APPROVED** [review]
Lgtm and tested

**Review by RomainValls — APPROVED** [review]

**Pull request merged**